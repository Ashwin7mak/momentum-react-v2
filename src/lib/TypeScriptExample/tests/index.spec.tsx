import React from 'react';
import { mount } from 'enzyme';
import { TypeScriptExample } from '..';

describe('tests for <TypeScriptExample />', () => {
  it('should match SnapShot', () => {
    const container = mount(<TypeScriptExample/>);

    expect(container).toMatchSnapshot();
  });
});
