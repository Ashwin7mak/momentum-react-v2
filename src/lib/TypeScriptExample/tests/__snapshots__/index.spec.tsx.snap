// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for <TypeScriptExample /> should match SnapShot 1`] = `
<TypeScriptExample
  param="test"
>
  <div>
    Hello this is a typescript component: 
    test
  </div>
</TypeScriptExample>
`;
